hie-plugin-api/Haskell/Ide/Engine/ModuleCache.hs

```diff
@@ -213,7 +212,7 @@ lookupCachedData fp tm info dat = do
 cacheModule :: FilePath -> (Either GHC.ParsedModule GHC.TypecheckedModule) -> IdeGhcM ()
 cacheModule uri modul = do
   uri' <- liftIO $ canonicalizePath uri
-  rfm <- GM.mkRevRedirMapFunc
+  rfm <- return id -- GM.mkRevRedirMapFunc
```

Comment:
Can we remove `revMap` from CachedInfo then?
https://github.com/haskell/haskell-ide-engine/blob/e1aeb180ab8065bd6a8413e050c6277191d0d33f/hie-plugin-api/Haskell/Ide/Engine/GhcModuleCache.hs#L44-L52